routes/web.php

```diff
@@ -51,6 +51,7 @@
 
 Route::get('/beatmapsets/search/{filters?}', ['as' => 'beatmapsets.search', 'uses' => 'BeatmapsetsController@search']);
 Route::resource('/beatmapsets', 'BeatmapsetsController', ['only' => ['index', 'show']]);
+Route::post('/beatmapsets/{beatmapset}/update-favorite', ['as' => 'beatmapsets.update-favorite', 'uses' => 'BeatmapsetsController@updateFavorite', 'middleware' => 'auth']);
```

Comment:
I think most of other routes define their middleware in controller. Additionally, `priv_check`'s `ensureLoggedIn` pretty much does the same thing.